packages/sanity/src/core/i18n/types.ts

```diff
@@ -137,6 +136,42 @@ export interface LocaleDefinition {
    */
   title: string
 
+  /**
+   * Week information for this locale. Based on the `Intl.Locale['weekInfo']` type.
+   */
+  weekInfo: LocaleWeekInfo
+}
+
+/**
+ * An object representing week information associated with the Locale data specified in
+ * {@link https://www.unicode.org/reports/tr35/tr35-dates.html#Date_Patterns_Week_Elements | UTS 35's Week Elements }
+ *
+ * @public
+ */
+export interface LocaleWeekInfo {
+  /**
+   * An integer indicating the first day of the week for the locale. Can be either 1 (Monday) or 7 (Sunday).
+   */
+  firstDay: 1 | 7
```

Comment:
what happens if another number besides 1 or 7 is provided? would it be better to have these as strings e.g. `firstDay: 'monday' | 'sunday'`?

Reply:
> what happens if another number besides 1 or 7 is provided? would it be better to have these as strings e.g. `firstDay: 'monday' | 'sunday'`?

Could maybe validate it in the `defineLocale` call? This is just following the `Intl` types. While I agree that spelling it out seems easier to read, I want to automate this information, so I'll stick with what that API gives us.